packages/rum-core/src/domain/tracing/tracer.spec.ts

```diff
@@ -391,6 +527,13 @@ describe('TraceIdentifier', () => {
 
     expect(traceIdentifier.toDecimalString()).toMatch(/^\d+$/)
   })
+
+  it('should pad the string to 16 characters', () => {
+    const traceIdentifier: any = new TraceIdentifier() // Forcing as any to access private member: buffer
+    traceIdentifier.buffer = new Uint8Array([0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07])
+
+    expect((traceIdentifier as TraceIdentifier).toPaddedHexadecimalString()).toEqual('0001020304050607')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** cast only when needed so you don't have to cast it back
```suggestion
    const traceIdentifier = new TraceIdentifier()
    // Forcing as any to access private member: buffer
    ;(traceIdentifier as any).buffer = new Uint8Array([0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07])

    expect(raceIdentifier.toPaddedHexadecimalString()).toEqual('0001020304050607')
```

Reply:
👌